lib/src/widgets/calendar_core.dart

```diff
@@ -95,6 +99,14 @@ class CalendarCore extends StatelessWidget {
               child: dayBuilder(context, day, baseDay),
             );
           },
+          dowHeight: dowHeight,
+          weekNumberVisible: weekNumbersVisible,
+          weekNumberBuilder: (context, day) {
+            return SizedBox(
+              height: rowHeight,
```

Comment:
This should be `height: constrainedRowHeight ?? rowHeight,` to work properly with `shouldFillViewport`.